elm.json

```diff
@@ -3,7 +3,7 @@
     "name": "elm-explorations/webgl",
     "summary": "Functional rendering with WebGL in Elm",
     "license": "BSD-3-Clause",
-    "version": "1.1.0",
+    "version": "1.2.0",
```

Comment:
I wouldn't include this change in this PR, as you want to leave the decision of when to bump the version to the author, as it might make sense to bundle another minor version bump here as well.